rllib/core/optim/tests/test_rl_optimizer_torch.py

```diff
@@ -308,16 +311,9 @@ def get_state(self) -> Mapping[str, Any]:
         }
 
 
-class TestRLOptimizer(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls) -> None:
-        ray.init()
-
-    @classmethod
-    def tearDownClass(cls) -> None:
-        ray.shutdown()
-
+class TestRLOptimizerTorch(unittest.TestCase):
```

Comment:
All these changes here seem to have slipped in from an earlier commit of your previous PR (RLOptimizer one). Can you make sure this is synched properly with master? Thx! :)